feature-libs/storefinder/components/store-finder-search/store-finder-search.component.html

```diff
@@ -27,7 +27,7 @@
         ></cx-icon>
         <cx-icon
           *cxFeatureLevel="'5.2'"
-          tabindex="-1"
+          [attr.tabindex]="(queryInput.value && queryInput.value.length) ? 0 : -1"
```

Comment:
I believe this is allowed in recent versions of Angular

```suggestion
          [attr.tabindex]="(queryInput.value?.length) ? 0 : -1"
```

Reply:
i will update the code and it already has feature level check